File: Cpp/282A.cpp
/*
* Link for the problem statement
* http://codeforces.com/problemset/problem/282/A
*/

#include<bits/stdc++.h>
using namespace std;
typedef long long int INT;

int main()
{
	int n;
	cin>>n;
	string s;
	int ans=0;
	while(n--)
	{
		cin>>s;
		switch(s[1])
		{
			case '+': ans++;
					  break;
			case '-': ans--;
					  break;
		}
	}
	cout<<ans;
	return 0;
}

File: Python/282A.py
'''
* Link for the problem statement
* http://codeforces.com/problemset/problem/282/A
'''

n=int(input().strip())
ans=0
for i in range(n):
	s=input()
	if s[1]=='+':
		ans+=1
	else:
		ans-=1
print(ans)
